datamol/utils/__init__.py

```diff
@@ -6,3 +6,6 @@
 from . import perf
 
 from . import decorators
+
+from .file_load_save import open_dataframe
+from .file_load_save import save_dataframe
```

Comment:
Can you rename the file to `file_io` please?

Reply:
Thinking twice about it. That module can't really be seen as a util module since it depends on a main datamol module `datamol.io.read_sdf`.

I am trying to better rationalize the datamol module hierarchy. Can you move all those functions to `datamol.io` instead please?